core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1776,6 +1776,33 @@ object TestUtils extends Logging {
         45000)
   }
 
+
+  def waitAndVerifyAcl(expected: AccessControlEntry,
+                       authorizer: JAuthorizer,
+                       resource: ResourcePattern,
+                       accessControlEntryFilter: AccessControlEntryFilter = AccessControlEntryFilter.ANY): Unit = {
```

Comment:
The only caller never specifies this argument, maybe we can remove it? Same below in `waitAndVerifyRemovedAcl()`